openedx/core/djangoapps/credentials/tasks/v1/tasks.py

```diff
@@ -43,12 +43,16 @@
     CertificateStatuses.downloadable,
 ]
 
-# Maximum number of retries before giving up. For reference, 11 retries with exponential backoff yields a maximum
-# waiting time of 2047 seconds (about 30 minutes). Setting this to None could yield unwanted behavior: infinite retries.
-MAX_RETRIES = 11
 
-
-@shared_task(bind=True, ignore_result=True)
+@shared_task(
+    bind=True,
+    ignore_result=True,
+    autoretry_for=(Exception,),
+    max_retries=10,
```

Comment:
Total number that each individual task will be attempted is `11` -- initial attempt + 10 retries.

This change updates the time period before an initial retry. Before it was 1s, now it will be "some number" between 0 - 30 (because we are also going to use Celery's jittering functionality to stagger retried tasks). 

The backoff will double each time to a maximum of 600s (10 minutes). 

While I believe `retry_jitter` is enabled by default, I added it as an argument for verbosity.